spark/src/main/resources/R/zeppelin_sparkr.R

```diff
@@ -31,7 +31,7 @@ print(paste("LibPath ", libPath))
 library(SparkR)
 
 
-SparkR:::connectBackend("localhost", port, 6000)
+SparkR:::connectBackend("localhost", port)
```

Comment:
@felixcheung 
`connectBackend <- function(hostname, port, timeout) { ... } ` doesn't specify a default value for parameter `timeout`, so any call made outside Spark need to specify it, overriding the parameters passed by user in `spark-env.sh` or in `spark-defaults.conf`.